crates/ruff/src/rules/flake8_bugbear/rules/function_uses_loop_variable.rs

```diff
@@ -76,9 +70,10 @@ where
                 self.names.extend(
                     visitor
                         .loaded
-                        .iter()
+                        .into_iter()
                         .filter(|(id, ..)| !arg_names.contains(id)),
                 );
+                return;
```

Comment:
This is the real fix. We don't want to walk the function node again because that results in visiting the argument initializers (and a lot of unnecessary visiting)